staging/src/k8s.io/apiserver/pkg/server/filters/maxinflight.go

```diff
@@ -48,15 +48,15 @@ func WithMaxInFlightLimit(
 	requestContextMapper apirequest.RequestContextMapper,
 	longRunningRequestCheck apirequest.LongRunningRequestCheck,
 ) http.Handler {
-	if nonMutatingLimit == 0 && mutatingLimit == 0 {
```

Comment:
Don’t tolerate bad input here. If we want a nicer error, check it before calling this function

Reply:
We did validate them in `ServerRunOptions.Validate`,  this can tolerate thirdparty using error.

Reply:
Just treat it as a public pkg function.

Reply:
It shouldn't tolerate it silently. It should fail, which it does today